cfme/scripting/quickstart.py

```diff
@@ -52,16 +52,20 @@ def args_for_current_venv():
 
 REQUIRED_PACKAGES = None
 INSTALL_COMMAND = None
+DEBUG_INSTALL_COMMAND = None
 
 if HAS_DNF:
     INSTALL_COMMAND = 'dnf install -y'
+    DEBUG_INSTALL_COMMAND = 'dnf debuginfo-install -y'
 elif HAS_YUM:
     INSTALL_COMMAND = 'yum install -y'
+    DEBUG_INSTALL_COMMAND = 'yum debuginfo-install -y'
 elif HAS_APT:
     INSTALL_COMMAND = 'apt install -y'
```

Comment:
there is a variable missing here

Reply:
No debuginfo install command for apt, looks like the python-dbg package can be installed as a normal package with apt.